src/functions/normalize_uploaded_files.php

```diff
@@ -24,6 +24,45 @@
  */
 function normalizeUploadedFiles(array $files)
 {
+    /**
+     * @param array $tmpNameTree
```

Comment:
Could you please document the types of these arrays?

Reply:
Will do 👍 This was mimicking the existing function (as they are multi-dimensional) - would you like me to update the existing one too?